app/views/mediated_pages/_SPEC-COLL_ineligible_notice.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class='<%= dialog_column_class %> success-page'>
+  <h1 id='dialogTitle'>
+    <h1>Sorry, we can't fulfill your request</h1>
+  </h1>
+
+  <p>Your SUNet ID indicates that you do not belong to any of the groups eligible for this service:</p>
+
+  <ul>
+    <li>faculty (including emeriti)</li>
+    <li>post-docs</li>
+    <li>graduate students</li>
+    <li>fellows</li>
+    <li>Stanford Visiting Scholars</li>
+  </ul>
+
+  <p>... with a current Stanford University card.</p>
+  <p>If you believe this is an error, contact <a href="mailto:greencirc@stanford.edu">greencirc@stanford.edu</a>.</p>
```

Comment:
I think I saw @saseestone bring this up, but should we send this to a SPEC email?